src/controllers/user.js

```diff
@@ -12,11 +12,38 @@ const getProfile = async (req, res) => {
         .json(currentUser);
     }
   } catch (error) {
-    // eslint-disable-next-line no-console
     return res.sendStatus(500).json({ message: error.message });
   }
 };
 
+const getUser = async (req, res) => {
+  try {
+    const foundUser = await UserModel.findById(req.params.id);
+    if (foundUser) {
+      const shownInfo = {
```

Comment:
Aren't you supposed to send every info and frontend decide on what to show? Why are you limited the info?  Because of the `passwordHash`?

Reply:
We need to restrict some information, for it not being shown on the website over some error or bug. Halit had told me to only return some info, even when a user registers ([can be found here](https://github.com/RCDD-202203-TUR-BEW/backend-capstone-turkey-lets-share/blob/9bba15b134d1070c092c67a2f7b9957601d40578/src/controllers/auth.js#L42-L50)).